<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Gate-Guard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="carDatabaseBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Authorized Vehicles  </string>
    </property>
   </widget>
   <widget class="QPushButton" name="activityHoursDbBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>590</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Manage Activity Hours</string>
    </property>
   </widget>
   <widget class="QPushButton" name="systemSettingsBtn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>590</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>System Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="activateBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>System Status: ON</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="lastEnteredTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>491</width>
      <height>361</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>371</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>License Plate</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phone Number</string>
     </property>
    </column>
   </widget>
   <widget class="QCommandLinkButton" name="refreshLastEnteredBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>510</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="FeedLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>451</width>
      <height>331</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>carDatabaseBtn</zorder>
   <zorder>activityHoursDbBtn</zorder>
   <zorder>systemSettingsBtn</zorder>
   <zorder>activateBtn</zorder>
   <zorder>lastEnteredTable</zorder>
   <zorder>FeedLabel</zorder>
   <zorder>refreshLastEnteredBtn</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
